src/viam/api/CMakeLists.txt

```diff
@@ -353,6 +487,8 @@ set_target_properties(
   VERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}
 )
 
+target_link_options(viamapi PRIVATE "LINKER:-z,defs")
```

Comment:
I have no clue about CMake. It's here because Drew used it to help debug linker problems, and should be removed now that we're done debugging. Sorry I can't explain more. :shrug: